thirdparty/DiskANN/src/linux_aligned_file_reader.cpp

```diff
@@ -9,13 +9,14 @@
 #include <sstream>
 #include "tsl/robin_map.h"
 #include "utils.h"
-#define MAX_EVENTS 1024
+#define MAX_EVENTS 128
```

Comment:
Why we still need it? To my understanding, we use maxnr from Knowhere to play its role right?